content/docs/getting_started/visual_story/setting_up@es.md

```diff
@@ -0,0 +1,49 @@
+---
+$title: Configuración
+$order: 0
+$parent: /content/docs/design/visual_story.md
```

Comment:
Please remove the added frontmatter from all the files. Only the translated $title should be present on translated files. Thank you.